sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,16 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self.args[0].is_real:
+            return True
```

Comment:
It is not necessary to solve inequalities (in the real or complex domain), since we have explicit formulas for the real and imaginary parts of cosh(x + iy) for real x and y.